### PR TITLE
Switch to BDB GATV port

### PR DESCRIPTION
**Pull request by Capkirk123**
Replace the FASA GATV docking port with the much nicer and more accurate looking BDB GATV docking port. Dimensions are nearly identical, so this shouldn't even break craft files.

### CODE REVIEW

File: GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg [hunk 1]
```diff
@@ -0,0 +1,156 @@
+PART
+{
+	name = bluedog_GATV_DockingPort
+	module = Part
+	author = CobaltWolf
+
+MODEL
+{
+   model = Bluedog_DB/Parts/Agena/bluedog_GATV_DockingPort
+}
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg:10]: Fix whitespace

File: GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	breakingForce = 50
+	breakingTorque = 50
+	maxTemp = 2000
+	fuelCrossFeed = false
+//	stagingIcon = DECOUPLER_VERT // Removed. The icon always displays when the ModuleJettison is present
+	bulkheadProfiles = size0p5, srf
+	stackSymmetry = 1
+
+	tags = agena belle 0.9375 9375 docking port target vehicle gatv
+  techtag = gatv
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg:67]: Fix whitespace

File: GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+	}
+
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = dockingPort_Lights
+		startEventGUIName = Lights On
+		endEventGUIName = Lights Off
+		actionGUIName = Toggle Lights
+    	defaultActionGroup = Light
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg:123]: whitespace

File: GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+		animSpeed = 10
+	}
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = dockingPort_RCS
+		startEventGUIName = RCS Indicator On
+		endEventGUIName = RCS Indicator Off
+		actionGUIName = Toggle RCS Indicator
+    	defaultActionGroup = RCS
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg:133]: whitespace

File: GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+		animSpeed = 10
+	}
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = dockingPort_SAS
+		startEventGUIName = SAS Indicator On
+		endEventGUIName = SAS Indicator Off
+		actionGUIName = Toggle SAS Indicator
+    	defaultActionGroup = SAS
```
Comment (DRVeyl) [inline GameData/ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort.cfg:143]: whitespace

File: GameData/ROCapsules/PartConfigs/Gemini/AgenaPort.cfg [hunk 6]
```diff
@@ -5,22 +5,24 @@ PART
...
-	  model = ROCapsules/Assets/FASA/Agena_Docking
-	  scale = 1.219, 1.219, 1.219
+		model = ROCapsules/Assets/BDB/GATV/bluedog_GATV_DockingPort
 	}
-	rescaleFactor = 1.0
-	scale = 1.219
-
+	rescaleFactor = 1.6
+	scale = 1.0
+
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Gemini/AgenaPort.cfg:17]: un-whitespace?

File: GameData/ROCapsules/PartConfigs/Gemini/AgenaPort.cfg [hunk 7]
```diff
@@ -73,71 +75,46 @@ PART
 		minDistanceToReEngage = 0.25 // 1.0
 		undockEjectionForce = 0.1 // 10
 	}
-
+
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Gemini/AgenaPort.cfg:78]: un-whitespace (excess tab/space at end of line)

**Comment by Capkirk123** [comment]
Fixed whitespace issues

**Pull request merged**